training/train.py

```diff
@@ -90,6 +90,15 @@ def generate_manifest_id_dict(manifest_list, package_id_dict):
     return manifest_id_dict
 
 
+def format_dict(package_id_dict, manifest_id_dict):
+    """Format the dictionaries."""
+    format_pkg_id_dict = [{'ecosystem': 'maven',
```

Comment:
I seriously don't understand why do we need an array list if its a single element?